_posts/2020-07-21-catalogs.md

```diff
@@ -8,6 +8,10 @@ authors:
   twitter: "dwysakowicz"
 ---
 
+In this blog post, I want to give a high level overview of catalogs in Flink. I will describe why should you consider using one and what can you achieve with it in place. I will also try to showcase how easy it is to use catalogs in the form of an end to end example that you can try out yourself.
+
+## Why do I need a catalog?
+
 With an ever-growing number of people working with data, it's a common practice for companies to build self-service platforms with the goal of democratizing their access across different teams and — especially — to enable users from any background to be independent in their data needs. In such environments, metadata management becomes a crucial aspect. Without it, users often work blindly, spending too much time searching for datasets and their location, figuring out data formats and similar cumbersome tasks.
```

Comment:
I'd probably place the intro sentence after this paragraph, then use the subtitle to kind of kick things off and transition to the actual blogpost. What do you think?

In general, it's nice to not speak in the first person in blogposts, so here's my proposal to slightly improve the sentence you wrote:

"In this blog post, we want to give you a high level overview of catalogs in Flink. We'll describe why you should consider using them and what you can achieve with one in place. To round it up, we'll also showcase how simple it is to combine catalogs and Flink, in the form of an end-to-end example that you can try out yourself."